build_tools/packaging/README.md

```diff
@@ -0,0 +1,96 @@
+# Building torchaudio packages for release
+
+ ## Anaconda packages
+
+ ### Linux
+
+ ```bash
+nvidia-docker run -it --ipc=host --rm -v $(pwd):/remote soumith/conda-cuda bash
+pushd remote/conda
+conda config --add channels pytorch
+conda config --add channels conda-forge
+yum -y install sox sox-devel llvm
```

Comment:
llvm?
that doesn't sound right

Reply:
also, `conda` has `sox` in `conda-forge` channel. add it as a dependency in `meta.yaml` and remove the `yum install sox` for the conda packages